iulianIachimov/src/main/java/iotask/utils/ReadFromDoc.java

```diff
@@ -0,0 +1,28 @@
+package iotask.utils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ReadFromDoc {
+  public static String[] readBufferedFile(File file) throws IOException {
```

Comment:
do you need an array of String? if not please change return to String, you could use StringWriter

Reply:
I added method countWordInFileFromString(String word, Path path) directly in Count class

Reply:
Sorry the method is countWordInFileFromString(String word, File file)